lxpanel-battery-plugin/README.md

```diff
@@ -0,0 +1,42 @@
+# cutiepi-lxpanel-battery
+
+lxpanel plugin for the CutiePi battery.
+
+[Original repo](https://github.com/BlackLight/cutiepi-lxpanel-battery)
```

Comment:
I think we can move this to copyright / credit section, 
not to confuse user who checks out this repo, what do you think?